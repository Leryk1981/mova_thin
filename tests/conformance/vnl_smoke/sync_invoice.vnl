emit channel="sync" payload='{"entity":"invoice","id":"inv_42","for":"cust_1","at":"2025-09-06T10:00:00Z"}'

