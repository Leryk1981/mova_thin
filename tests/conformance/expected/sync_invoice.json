{
  "mova_version": "3.3",
  "id": "sync_invoice",
  "description": "Generated from VNL",
  "policies": {
    "timeout_ms": 15000
  },
  "actions": [
    {
      "type": "emit_event",
      "channel": "sync",
      "payload": {
        "entity": "invoice",
        "id": "inv_42",
        "for": "cust_1",
        "at": "2025-09-06T10:00:00Z"
      }
    }
  ]
}